cmd/main.go

```diff
@@ -86,6 +86,8 @@ var Options struct {
 	HostStateMonitorInterval    time.Duration `envconfig:"HOST_MONITOR_INTERVAL" default:"8s"`
 	Versions                    versions.Versions
 	OpenshiftVersions           string        `envconfig:"OPENSHIFT_VERSIONS"`
+	ReleaseImageOverride        string        `envconfig:"OPENSHIFT_INSTALL_RELEASE_IMAGE"`
+	ReleaseImageMirror          string        `envconfig:"OPENSHIFT_INSTALL_RELEASE_IMAGE_MIRROR" default:""`
```

Comment:
Seems that ReleaseImageOverride and ReleaseImageMirror are treated the same, except for where you get the registry certificate.
I suggest having a single way to override the release image and a different flag for providing the mirror cert.

```suggestion
	mirrorCert          string        `envconfig:"OPENSHIFT_INSTALL_MIRROR_CERT" default:""`
```